app/models/lighthouse526_document_upload.rb

```diff
@@ -24,6 +24,12 @@ class Lighthouse526DocumentUpload < ApplicationRecord
   # Veteran Uploads must reference a FormAttachment record, where a Veteran-submitted file is stored
   validates :form_attachment, presence: true, if: :veteran_upload?
 
+  # Window for polling Lighthouse for the status of an upload
+  scope :status_update_required, lambda {
+                                   where('status_last_polled_at < ?', 1.hour.ago.utc)
```

Comment:
we should use arel instead of interpolation. it allows for more complex joins later. the arel version would be
`where(arel_table[: status_last_polled_at].lt(1.hour.ago.etc))`

Reply:
Revised as suggested in [5e74d15](https://github.com/department-of-veterans-affairs/vets-api/pull/17473/commits/5e74d15880ec77ab4e6ef6048fe48ee92810ba8e)